tools/parallel-testing/counters.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
This file is auto generated based on the testvectors, so it will chnage over time.
It should be placed in the `.gitignore`